tests/flow/declare_module_exports/flow-typed/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -42,12 +42,12 @@ declare module \\"declare_m_e_with_other_value_declares\\" {
 declare module \\"declare_m_e_with_other_type_declares\\" {
   declare module.exports: number;
   declare type str2 = string;
-}
-/**
+
+  /**
```

Comment:
The flow parser has completely busted parsing location for this one. It happened to do the right thing in the previous version by chance.

<img width="1019" alt="screen shot 2017-03-18 at 5 59 27 pm" src="https://cloud.githubusercontent.com/assets/197597/24077311/47fd4688-0c06-11e7-8d8d-a3b92c3087ef.png">